application/backend/src/main/resources/apps/bulma/components/buttons/.content.json

```diff
@@ -3,5 +3,5 @@
   "group": "Bulma.io Elements",
   "allowedComponents": ["/apps/bulma/components/button"],
   "sling:resourceType": "ws:Component",
-  "title": "Buttons list"
+  "title": "Buttons"
```

Comment:
According to the component filename change we have to update a template as well.